src/gui/src/UI/UIDesktop.js

```diff
@@ -710,6 +710,7 @@ async function UIDesktop(options){
         show_hidden_files: JSON.parse(await puter.kv.get('user_preferences.show_hidden_files')),
         language: await puter.kv.get('user_preferences.language'),
         clock_visible: await puter.kv.get('user_preferences.clock_visible'),
+        hide_desktop_icons: JSON.parse(await puter.kv.get('user_preferences.hide_desktop_icons')) ?? false,
```

Comment:
the `?? false` logic is redundant.

also, why do we need to `JSON.parse` the value coming from `puter.kv.get('user_preferences.hide_desktop_icons')`?
 shouldn't that always give us either a `Falsy` or Non falsy  value?